test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/SearchCommandTests.cs

```diff
@@ -860,10 +899,15 @@ public void SearchCommand_NoResultsFoundTest()
             using (MockServer server = new MockServer())
             using (SimpleTestPathContext config = new SimpleTestPathContext())
             {
-                CommandRunner.Run(
-                    nugetexe,
-                    config.WorkingDirectory,
-                    $"source add -name mockSource -source {server.Uri}v3/index.json -configfile {config.NuGetConfig}");
+                // Arrange the NuGet.Config file
+                string nugetConfigContent =
```

Comment:
I think the default nuget config has a lot more than just this configuration. 

Can you please use helpers such as: https://github.com/NuGet/NuGet.Client/blob/1845d6bd450a7453d573035371c9fec43683d1ef/test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestSettingsContext.cs#L222-L227.